lib/main_dev.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:chat_app/AppConfig.dart';
+import 'package:chat_app/firebase_options_dev.dart';
+import 'package:chat_app/main.dart';
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+void main() async {
```

Comment:
after Arham will merge the code, u will need to adjust and resolve things. so keep that in mind